startup.py

```diff
@@ -0,0 +1,340 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import os
+import sys
+import re
+import glob
+import subprocess
+import xml.etree.ElementTree as XML_ET
```

Comment:
I'm being nitpicky, but since there's only one place you're using this, mind doing `from xml.etree import ElementTree` and then calling `ElementTree.parse()` below?